README.md

```diff
@@ -167,6 +170,28 @@ from the `txn` table when `commit()` returned an error is undefined.
 
 [Back to TOC](#table-of-contents)
 
+## Directives
+
+### lmdb_encryption_key_data
+
+**syntax:** *lmdb_encryption_key_data "YourPassword";*
+
+**context:** *main*
+
+Encrypt the lmdb database, note that set lmdb_encryption_key_data to enable encryption.
+
+[Back to TOC](#table-of-contents)
+
+### lmdb_encryption_type
+
+**syntax:** *lmdb_encryption_type "AES-256-GCM";*
+
+**context:** *main*
+
+Set the lmdb database encryption mode. The default encryption mode is aes-256-gcm. The optional encryption modes are chacha20-poly1305 and aes-256-gcm. Note that lmdb_encryption_type needs to be set only when lmdb_encryption_key_data is set.
```

Comment:
The example shows `AES-256-GCM` (uppercase) while this doc string shows `aes-256-gcm` and `chacha20-poly1305` (lowercase). I think it would be good to make the casing consistent between the two so that things are completely unambiguous (even if the setting itself is case-insensitive).

Reply:
I agree with your opinion.